conf/char_athena.conf

```diff
@@ -172,11 +172,14 @@ char_del_delay: 86400
 
 // Restrict character deletion by email address or birthdate.
 // This restricts players from changing the langtype and deleting characters.
-// For birthdate, the client must be 20100803 or newer.
 // Defaults based on client date.
 // 1: Email address
 // 2: Birthdate
 // 3: Email address or Birthdate
+// IMPORTANT!
+// - This config only works for clients that send 0x0068 or 0x01fb for delete request.
+// - Try use langtype 1 for newer clients (2013+), to use 0x01fb.
+// - Clients that are not using 0x0068 or 0x01fb, only use birthdate (YYMMDD) as default.
```

Comment:
This might be a bit to technical for the comment here.
Maybe we/you should hand over the packet id to your newly created function chclif_delchar_check and include those checks in there.
For example:
The user has set his config to email address, but the client sends us packet that are not 0x0068 or 0x01fb, then you could add a warning which is displayed on the console.

As for birthdate in general we could add checks like we did for auction and battle conf in the mapserver, to check if the packet version is higher then 20100803 when the user has the option set to &2.